File: src/test/java/kitchenpos/menugroup/MenuGroupFixture.java
package kitchenpos.menugroup;

import kitchenpos.domain.MenuGroup;

import java.util.UUID;

public class MenuGroupFixture {
    public static MenuGroup menuGroup() {
        return new MenuGroup(UUID.randomUUID(), "추천메뉴");
    }

    public static MenuGroup menuGroup(String name) {
        return new MenuGroup(name);
    }
}

File: src/test/kotlin/kitchenpos/menugroup/MenuGroupFixture.kt
package kitchenpos.menugroup

import kitchenpos.domain.MenuGroup

class MenuGroupFixture {
    companion object {
        private const val DEFAULT_NAME: String = "테스트 메뉴 그룹"

        fun fixture(
            name: String? = DEFAULT_NAME
        ): MenuGroup {
            val menuGroup = MenuGroup()
            menuGroup.name = name
            return menuGroup
        }
    }
}
